src/test/javascript/spec/component/shared/exercise-scores-export-button.component.spec.ts

```diff
@@ -0,0 +1,96 @@
+import { ComponentFixture, TestBed, fakeAsync, tick } from '@angular/core/testing';
```

Comment:
`fakeAsync` and `tick` are unused.

Reply:
Ups.. missed that :slightly_smiling_face: